pkg/cmd/extension/manager_test.go

```diff
@@ -872,34 +936,48 @@ func TestManager_repo_not_found(t *testing.T) {
 
 func TestManager_Create(t *testing.T) {
 	chdirTemp(t)
+	err := os.MkdirAll("gh-test", 0755)
```

Comment:
Manually create the extension directory now, but we have assertions that `git init` is executed which will do this folder creation in production.